charts/datadog/values.yaml

```diff
@@ -308,6 +308,22 @@ datadog:
     # datadog.apm.hostSocketPath -- Host path to the trace-agent socket
     hostSocketPath: /var/run/datadog/
 
+  # datadog.otlp -- OTLP ingest related configuration
+  ## Set a protocol section to enable its default endpoint.
+  otlp: {}
```

Comment:
I'm still in favour of defining the full configuration structure with default value.

```yaml
datadog:
  otlp:
    enabled: false
    receiver:
      grpc:
        enabled: true
        port: 4317
      http:
        enabled: true
        port: 4318
```

With this approach, the user will only have to set `datadog.otlp.enabled: true` to have it working. which is convenient for user that use the `--set` option with the `helm install` command. (ex: `helm install foo --set datadog.otlp.enabled=true datadog/datadog`

Reply:
Changed to have an `enabled` option for each protocol.